src/components/card/Card.js

```diff
@@ -17,33 +19,106 @@ function tellHouseColor(hogwartsHouse) {
 
 function Card(props) {
   const houseColor = tellHouseColor(props.hogwartsHouse);
+  // const showDetails = true;
+  const [showDetails, setShowDetails] = useState(false);
+  const [emoji, setEmoji] = useState("");
 
   return (
     <section className="section-card">
       <div className="section-header">
         <div>
           <img className="section-img" src={props.imageURL} alt="" />
         </div>
-        <h2 className="section-header-title">{props.characterName}</h2>
+        <h2 className="section-header-title">
+          {emoji}
+          {props.characterName}
+        </h2>
         <div className={houseColor}>{props.hogwartsHouse}</div>
-        <button className="expand-more-button">
-          <img className="expand-more-img" src={expandMore} alt="" />
+        <button
+          className={!showDetails ? "expand-more-button" : "expand-less-button"}
+          onClick={() => {
+            setShowDetails(!showDetails);
+          }}
+        >
+          <img
+            className="expand-more-img"
+            src={!showDetails ? expandMore : expandLess}
+            alt=""
+          />
         </button>
       </div>
-      <div className="section-content">
-        <ul className="main-info-list">
-          <h3>Basics:</h3>
-          <li>Gender: {props.gender}</li>
-          <li>Birthday: {props.birthday}</li>
-          <li>Patronus: {props.patronus}</li>
-        </ul>
-        <ul className="wand-list">
-          <h3>Wand:</h3>
-          <li>Wood: {props.wandWood}</li>
-          <li>Core: {props.wandCore}</li>
-          <li>Length: {props.wandLength}</li>
-        </ul>
-      </div>
+      {showDetails && (
+        <div className="section-content">
+          <div className="content-text">
+            <ul className="main-info-list">
+              <h3>Basics:</h3>
```

Comment:
Only `li` elements are valid children of lists in HTML! The `h3` should be outside of the `ul`. You can also check out [description lists](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl)

Reply:
Ok! Fixed this. :-)